addons/dockable_container/plugin.gd

```diff
@@ -3,13 +3,14 @@ extends EditorPlugin
 
 const DockableContainer = preload("dockable_container.gd")
 const LayoutInspectorPlugin = preload("inspector_plugin/editor_inspector_plugin.gd")
+const Icon = preload("res://addons/dockable_container/icon.svg")
```

Comment:
This `preload` should use the relative path `icon.svg` instead of the absolute path `res://...`, to match all other uses of `preload` in this code base. This avoids the icon not being found in case the plugin is not installed in its default path. I know this will most likely never be the case, but it doesn't hurt to make things less dependent on environment.

Reply:
Thanks for pointing that out, slipped out of my mind. I've updated it now along with the README.